core/cache/src/main/java/uk/gov/gchq/gaffer/cache/ICacheService.java

```diff
@@ -59,8 +57,9 @@ public interface ICacheService {
      * @param <K>       The object type that acts as the key for the cache
      * @param <V>       The value that is stored in the cache
      * @return the requested cache object
+     * @throws CacheOperationException if issue getting from cache.
      */
-    default <K, V> V getFromCache(final String cacheName, final K key) {
+    default <K, V> V getFromCache(final String cacheName, final K key) throws CacheOperationException {
         final ICache<K, V> cache = getCache(cacheName);
         return cache.get(key);
```

Comment:
Is this throwing an exception now, or did you mean to roll this back. Also the two lines can be condensed into one as you've done with the other methods.

Reply:
So I think this throwing an exception is possibly correct as the `getFromCache` method in the `Cache` class already had this declaration on which actually just calls this method e.g. the services `getFromCache` by default. Adding it here makes it consistent across the classes and is probably correct that it is an exception.